packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/addDevice/add-device.html

```diff
@@ -0,0 +1,105 @@
+            <div class="row">
+                <oui-field
+                    class="col-md-12"
+                    data-help-text="{{:: 'telephony_line_softphone_generate_warning' | translate }}"
+                    label="{{ ::'telephony_line_softphone_add_device_label' | translate }}"
+                    size="sm"
+                >
+                    <div class="input-group">
+                        <input
+                            type="text"
+                            id="input_name"
+                            name="input_name"
+                            class="oui-input"
+                            data-ng-model="$ctrl.input.name"
+                            data-ng-pattern="/^[a-zA-Z0-9\-\_]+$/"
+                            data-ng-class="{'oui-field_error': $ctrl.softphoneAddDeviceForm.input_name.$invalid}"
+                            data-translate-attr="{ placeholder: 'telephony_line_softphone_device_name_placeholder' }"
+                            required
+                        />
+                        <div class="input-group-btn">
+                            <oui-button
+                                on-click=":: $ctrl.checkInputAndGenerateLink()"
+                                data-variant="primary"
+                            >
+                                <span
+                                    data-translate="telephony_line_softphone_generate_link"
+                                >
+                                </span>
+                            </oui-button>
+                        </div>
+                    </div>
+                </oui-field>
+            </div>
+            <div class="row">
+                <oui-field
+                    class="col-md-12"
+                    label="{{ ::'telephony_line_softphone_record_link' | translate }}"
+                    size="sm"
+                >
+                    <div class="input-icons-softphone-record-link">
+                        <input
+                            type="text"
+                            id="input_record_link"
+                            name="input_record_link"
+                            data-ng-readonly="true"
+                            class="oui-input"
+                            required
+                            data-ng-model="$ctrl.input.recordLink"
+                        />
+                        <i
+                            class="oui-icon oui-icon-copy-normal"
+                            aria-hidden="true"
+                        >
+                        </i>
+                    </div>
+                </oui-field>
+            </div>
+            <div class="row">
+                <oui-message data-type="info">
+                    <span
+                        data-translate="telephony_line_softphone_enroll_device_theme_description"
+                    ></span>
+                    <br />
+                    <a href="">
```

Comment:
```suggestion
                    <a href="">
```
You can put the href of page with modal closed or change for a button for to do a onClick goBack for close the modal

Reply:
I am going to check if I have the link but I did not find it, gonna recheck